### PR TITLE
Add test of BusBuildersFromConfig

### PR DESCRIPTION
**Pull request by tyx**
I guess phpUnit assertions could be improved... but I have many difficulties to find what I want.

### CODE REVIEW

File: tests/DependencyInjection/Compiler/BusBuildersFromConfig.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            ],
+        ]);
+
+        $this->assertEquals(
+            new BusBuilder('default', 'other.inflector', ['my.middleware']),
+            $builders->getIterator()['default']
+        );
+    }
+
+    public function test method inflector of each bus can be overrided()
```
Comment (rosstuck) [inline tests/DependencyInjection/Compiler/BusBuildersFromConfig.php]: First time I've ever seen this in the wild. I'm....not sure about this. 😑

File: src/DependencyInjection/Compiler/BusBuilder/BusBuilders.php [hunk 2]
```diff
@@ -23,6 +23,7 @@ public function __construct(array $busBuilders, string $defaultBusId)
             $this->busBuilders[$builder->id()] = $builder;
         }
 
+        $this->assertValidBusId($defaultBusId);
```
Comment (rosstuck) [inline src/DependencyInjection/Compiler/BusBuilder/BusBuilders.php]: Good catch.

**Comment by rosstuck** [comment]
Solid PR, thanks a ton! I'm not sure I'm ready to submit to the majesty of non-breaking spaces in method names, maybe revert those and I promise a swift and decisive merge! 🥇

File: tests/DependencyInjection/Compiler/BusBuildersFromConfig.php (hunk 1 above)
Comment (tyx) [reply tests/DependencyInjection/Compiler/BusBuildersFromConfig.php]: oh did not realize, I did it mechanically ! My bad, i will update with `_` :+1:

**Comment by tyx** [comment]
I removed the unbreakable spaces of course !

**Comment by rosstuck** [comment]
Thanks a ton! :)

**Pull request merged**